pydantic/networks.py

```diff
@@ -351,6 +393,114 @@ class PostgresDsn(AnyUrl):
     }
     user_required = True
 
+    __slots__ = ('hosts',)
```

Comment:
are you sure this is right? Should we not do something like `AnyUrl.__slots__ + ('hosts',)`?